src/app/components/Containers/TextContainer.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Headline from '../Headline';
+import SubHeading from '../SubHeading';
+import {
+  commonTextPropTypes,
+  baseTextDefaultPropTypes,
+} from '../../models/proptypes';
+
+const Blocks = {
+  headline: Headline,
+  subheading: SubHeading,
+};
+
+const TextContainer = blocks => {
+  const extractType = blocks.type;
+  const passingText = blocks.model.blocks[0].model.blocks[0].model;
```

Comment:
This constant name is slightly misleading, since it's actually the model object. It would be better if you had `const { text }  = blocks.model.blocks[0].model.blocks[0].model;` and just updated `BlockType` to take in text instead of the model.

Reply:
This would also mean that you can remove the `containerText` function from `model/blocks.js`, since you can just pass in the text string, rather than the object.